_includes/sidebar.html

```diff
@@ -38,17 +38,18 @@
       {% endfor %}
     </nav>
 
-    <p>
-      {% if page.lang == site.default_lang %}
-        {% assign url = page.url %}
-      {% else %}
-        {% assign url = page.url | slice: 3, 99 %}
-      {% endif %}
-
-      {% for lang in site.languages %}
-        <a href="{% if lang == site.default_lang %} {{url}} {% else %} /{{lang}}{{url}} {% endif %}">{{lang}}</a>
-      {% endfor %}
-    </p>
+    <div id="languages">
+      {% assign url = page.url | slice: 3, 99 %}
```

Comment:
i mean, remove the first and then join again, but i cannot figure out how to remove the first item of an array